src/commands/cmd_list.cc

```diff
@@ -154,6 +154,79 @@ class CommandRPop : public CommandPop {
   CommandRPop() : CommandPop(false) {}
 };
 
+class CommandLMPop : public Commander {
+ public:
+  /// format: LMPOP #numkeys key0 [key1 ...] <LEFT | RIGHT> [COUNT count]
+  Status Parse(const std::vector<std::string> &args) override {
+    CommandParser parser(args, 1);
+
+    auto num_keys = GET_OR_RET(parser.TakeInt<uint32_t>());
+    keys_.clear();
```

Comment:
Is this call redundant?

Reply:
This call is some kind of "defensive programming". If the Parse() function is called more than once, the keys_ vector would has too many args (accumulates a batch each time called). 
Besides, the following Execute() relies on keys_.size(), so we don't want the keys_ vector to be polluted. 
Lastly, this calling has almost no overhead. 
So I tend to leave it there.